### DISCUSSION TITLE
Any plans about other language bindings?

### DISCUSSION THREAD

**Discussion by ffunatsu** [bevyengine/bevy#11465] [Q&A] [answered]
I wonder that bevy engine can be used from other languages more than Rust, just like `wgpu-native` crate for `wgpu`.
Are there any official plans or third party portings to other languages like C++ or JavaScript, etc?

(Sorry if it's duplicate Q&A. If the issue already exists, I'd like to check it)

**Comment by nicopap** [comment, marked as answer]
Currently, bevy doesn't plan to maintain non-rust language bindings.

Third party crates are free to create bindings for other languages though.

See this issue for a discussion why: https://github.com/bevyengine/bevy/issues/2352
